js/form.js

```diff
@@ -0,0 +1,20 @@
+//  Form Values  //
+const form = document.getElementById("checkout-form");
+const submitButton = document.getElementById("submit");
+
+let formInfo = [];
+
+function submitForm() {
+  let formInfo = []; //testing purposes//
```

Comment:
let variables are block scoped. your code will attempt to use the formInfo declared inside the function first.